.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+## Description
+
+Please include a summary of the changes made.
+
+### Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes a bug)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] feature Improvement (changes in an existing feature that improves it)
+- [ ] Breaking change (big change to existing code like re-design)
+- [ ] Clean (changes only to names/format/order and stuff like it)
```

Comment:
why is this a checkbox?

Reply:
So u can mark the right things

Reply:
I think you are misusing this feature. you are moving the labels describing what the PR is instead of moving other information which are less relevant looking from outside the PR.